README.md

```diff
@@ -184,9 +74,70 @@ python mava/systems/ff_ippo.py env=rware env/scenario=tiny-4ag
+    </a>
+    <a href="docs/images/benchmark_results/lbf.png">
+        <img src="docs/images/benchmark_results/lbf.png" alt="Mava performance across 7 Level Based Foraging environments" width="30%" style="display:inline-block; margin-right: 10px;"/>
+    </a>
+    <a href="docs/images/benchmark_results/smax.png">
+        <img src="docs/images/benchmark_results/smax.png" alt="Mava performance across 11 Smax environments" width="30%" style="display:inline-block; margin-right: 10px;"/>
+    </a>
+    <a href="docs/images/benchmark_results/connector.png">
+        <img src="docs/images/benchmark_results/connector.png" alt="Mava performance across 4 Conneector environments" width="30%" style="display:inline-block; margin-right: 10px;"/>
+    </a>
+    <a href="docs/images/benchmark_results/mabrax.png">
+        <img src="docs/images/benchmark_results/mabrax.png" alt="Mava performance across 5 MaBrax environments" width="30%" style="display:inline-block; margin-right: 10px;"/>
+    </a>
+    <a href="docs/images/benchmark_results/mpe.png">
+        <img src="docs/images/benchmark_results/mpe.png" alt="Mava performance across 3 Multi-Particle environments" width="30%" style="display:inline-block; margin-right: 10px;"/>
+    </a>
+    <br>
+    <a href="docs/images/benchmark_results/legend.jpg">
+        <img src="docs/images/benchmark_results/legend.jpg" alt="Legend" width="60%" style="display:inline-block; margin-right: 10px;"/>
+    </a>
+    <div style="text-align:center; margin-top: 10px;"> <strong>Mava's algorithm performance:</strong> Each algorithm was tuned for 40 trials with the TPE optimizer and benchmarked over 10 seeds for each scenario. Environments from top left Robot Warehouse (aggregated over 15 scenarios) Level Based Foraging (aggregated over 7 scenarios) SMAX (aggregated over 11 scenarios) Connector (aggregated over 4 scenarios) MaBrax (aggregated over 5 scenarios) Multi-Particle Env (aggregated over 3 scenarios)</div>
+</p>
+
+## Code Philosophy 🧘
+
+The current code in Mava is adapted from [PureJaxRL][purejaxrl] which provides high-quality single-file implementations with research-friendly features. In turn, PureJaxRL is inspired by the code philosophy from [CleanRL][cleanrl]. Along this vein of easy-to-use and understandable RL codebases, Mava is not designed to be a modular library and is not meant to be imported. Our repository focuses on simplicity and clarity in its implementations while utilising the advantages offered by JAX such as `pmap` and `vmap`, making it an excellent resource for researchers and practitioners to build upon. A noteable difference between Mava and CleanRL is that Mava creates small utilities for heavily re-used elements, such as networks and logging, we've found that this, in addition to hydra configs, greatly improves the readability of the algorithms.
```

Comment:
Is this still the case? Or has the code diverged significantly? If so, I think we can say something like "the original code was closely built off of purejaxrl".

Reply:
Same question to consider regarding cleanrl

Reply:
Changed slightly to say the code was originally adapted from PureJaxRL.